content/webapp/views/components/HTMLSerializers/index.tsx

```diff
@@ -107,12 +111,33 @@ export const defaultSerializer: JSXFunctionSerializer = (
 
       const isInPage = linkUrl.match(/^https:\/\/(#.*)/i);
       const hashLink = isInPage && isInPage[1];
+      const isWorkLink = linkUrl.match(
+        /^https:\/\/wellcomecollection.org\/works/i
+      );
 
       const fileExtension = linkUrl.match(/\.[0-9a-z]+$/i);
 
       const documentType =
         fileExtension && fileExtension[0].substring(1).toUpperCase();
 
+      if (isWorkLink) {
+        return (
+          <div
+            data-id="work-link-component"
```

Comment:
My [change](https://github.com/wellcomecollection/wellcomecollection.org/pull/12120/files) isn't in yet but it's now `data-gtm-id`

Reply:
Possibly another argument for the centralised component!